src/Uno.Templates/content/unoapp/MyExtensionsApp._1.Mobile/MyExtensionsApp._1.Mobile.csproj

```diff
@@ -274,4 +274,10 @@
     <!--#endif-->
   </ItemGroup>
   <Import Project="..\MyExtensionsApp._1.Shared\base.props" />
+
+  <Target Name="ValidateOverrides" BeforeTargets="Restore;_CheckForUnsupportedTargetFramework" Condition="'$(OverrideTargetFramework)' != ''">
+    <Error
+        Text="OverrideTargetFramework set to '$(OverrideTargetFramework)' is missing valid target. Set OverrideTargetFramework to $([MSBuild]::Escape('$'))(DotNetVersion)-ios, $([MSBuild]::Escape('$'))(DotNetVersion)-android or $([MSBuild]::Escape('$'))(DotNetVersion)-maccatalyst or skip building this project (eg unload the project in Visual Studio)"
```

Comment:
We just need to dynamically set this based on the heads that are created so we don't confuse people with reference to project types that aren't actually supported...